docs/technical-documentation/alpaca_tips.md

```diff
@@ -0,0 +1,38 @@
+# Alpaca Tips
+
+[Alpaca](https://github.com/Islandora/Alpaca) is event-driven middleware based on [Apache Camel](https://camel.apache.org/) for Islandora
+
+Currently, Alpaca ships with four event-driven components which are being run with [Apache Karaf](https://karaf.apache.org/)
+- [islandora-connector-derivative](#islandora-connector-derivative)
+- [islandora-http-client](#islandora-http-client)
+- [islandora-indexing-fcrepo](#islandora-indexing-fcrepo)
+- [islandora-indexing-triplestore](#islandora-indexing-triplestore)
+
+## islandora-connector-derivative
+This service receives requests from Drupal when it wants to create derivatives and passes that request along to a microservice in [Crayfish](https://github.com/Islandora/Crayfish). When it receives the derivative file back from the microservice, it passes the file back to Drupal.
+
+## islandora-http-client
+
+## islandora-indexing-fcrepo
+This service receives requests from Drupal when a node is created, edited, or deleted, and when a media is created or edited, and when an item is created, edited or deleted. These requests are passed along to [Milliner](https://github.com/Islandora/Crayfish/tree/dev/Milliner) microservice in [Crayfish](https://github.com/Islandora/Crayfish) to convert Drupal entities into Fedora resources and communicate with Fedora (via [Chullo](https://github.com/Islandora-CLAW/chullo)).
```

Comment:
You could shrink that first sentence to "This service receives requests from Drupal in response to write operations on entities."